FSQLocationBroker/FSQLocationBroker.m

```diff
@@ -26,7 +26,7 @@ @interface FSQLocationBroker ()
 @property (atomic, readwrite) NSSet *locationSubscribers;
 @property (atomic, readwrite) NSSet *regionSubscribers;
 @property (atomic, readwrite) NSSet *visitSubscribers;
-@property (nonatomic, copy) CLLocation *currentLocation;
+@property (atomic, copy) CLLocation *currentLocation;
```

Comment:
the header had atomic and the .m file has nonatomic, it was causing a warning and I thought that was weird